sdks/python/container/run_validatescontainer.sh

```diff
@@ -115,6 +115,6 @@ python setup.py nosetests \
 
 # Delete the container locally and remotely
 docker rmi $CONTAINER:$TAG
-gcloud container images delete $CONTAINER:$TAG --quiet
```

Comment:
My sense is that for this script we might not want `--quiet`?

Reply:
The --quiet also suppresses the Yes/No prompt, but works without it because Yes is the default. Removed the --quiet